pkg/infra/metrics/metrics.go

```diff
@@ -104,6 +105,27 @@ var (
 
 	// MRenderingQueue is a metric gauge for image rendering queue size
 	MRenderingQueue prometheus.Gauge
+
+	// MAccessRoleCount is a metric gauge for total number of roles
+	MAccessRoleCount prometheus.Gauge
+
+	// MAccessBuiltinRoleCount is a metric gauge for number of role assignments to teams
```

Comment:
It seems the comment is off as it talks about `MAccessBuiltinRoleCount` ? 🤔 Or if I get this right, we actually need `MAccessBuiltinRoleCount` as we won't be releasing assignments to teams.

Reply:
Will people be changing the number of builtin roles?  I removed it because I thought they weren't going to change, but I can add it back easily enough